### PR TITLE
Events / subject tracking RJSF simplifications

### LINKED ISSUE #388: Simplify events + patrols RJSF with advancedfields, inline defaults, and task groups
This is now, if not complete, at least in a reviewable state for patrols, following #386.

Now we need to do the same for events and subject-tracking.

towards https://github.com/wildlife-dynamics/compose/issues/131

### PR DESCRIPTION
**Pull request by cisaacstern**
closes #388

### CODE REVIEW

File: examples/events/spec.yaml [hunk 1]
```diff
@@ -64,6 +75,15 @@ workflow:
     partial:
       geo_layers:
         - ${{ workflow.events_map_layer.return }}
+      tile_layers:
+        - name: "SATELLITE"
```
Comment (walljcg) [inline examples/events/spec.yaml:79]: should be: 
-name: "TERRAIN"
-name: "SATELLITE"
   opacity: 0.5

File: examples/events/spec.yaml [hunk 2]
```diff
@@ -133,6 +159,15 @@ workflow:
     partial:
       geo_layers:
         - ${{ workflow.fd_map_layer.return }}
+      tile_layers:
+        - name: "SATELLITE"
```
Comment (walljcg) [inline examples/events/spec.yaml:163]: -name: "TERRAIN"
-name: "SATELLITE"
   opacity: 0.5

File: examples/events/spec.yaml [hunk 3]
```diff
@@ -251,6 +306,16 @@ workflow:
   - name: Draw Ecomap from Feature Density
     id: grouped_fd_ecomap
     task: draw_ecomap
+    partial:
+      tile_layers:
+        - name: "SATELLITE"
```
Comment (walljcg) [inline examples/events/spec.yaml:311]: -name: "TERRAIN"
-name: "SATELLITE"
   opacity: 0.5

File: examples/subject-tracking/spec.yaml [hunk 4]
```diff
@@ -97,6 +116,16 @@ workflow:
   - name: Draw Ecomaps for each trajectory group
     id: traj_ecomap
     task: draw_ecomap
+    partial:
+      tile_layers:
```
Comment (walljcg) [inline examples/subject-tracking/spec.yaml:120]: -name: "TERRAIN"
-name: "SATELLITE"
   opacity: 0.5

File: examples/subject-tracking/spec.yaml [hunk 5]
```diff
@@ -144,6 +177,16 @@ workflow:
   - name: Draw Ecomaps for each trajectory group
     id: traj_daynight_ecomap
     task: draw_ecomap
+    partial:
+      tile_layers:
+        - name: "SATELLITE"
```
Comment (walljcg) [inline examples/subject-tracking/spec.yaml:182]: -name: "TERRAIN"
-name: "SATELLITE"
   opacity: 0.5

File: examples/subject-tracking/spec.yaml [hunk 6]
```diff
@@ -158,6 +201,8 @@ workflow:
   - name: Create Map Widgets for Trajectories
     id: traj_map_daynight_widgets_sv
     task: create_map_widget_single_view
+    partial:
+      title: "Subject Group Day/Night Map"
```
Comment (walljcg) [inline examples/subject-tracking/spec.yaml:205]: title: "Subject Group Night/Day Map"

File: examples/subject-tracking/spec.yaml [hunk 7]
```diff
@@ -246,6 +307,8 @@ workflow:
   - name: Create Single Value Widgets for Day/Night Ratio Per Group
     id: daynight_ratio_sv_widgets
     task: create_single_value_widget_single_view
+    partial:
+      title: "Day Night Ratio"
```
Comment (walljcg) [inline examples/subject-tracking/spec.yaml:311]: Night/Day Ratio

File: examples/subject-tracking/spec.yaml [hunk 8]
```diff
@@ -337,6 +418,16 @@ workflow:
   - name: Draw Ecomap from Time Density
     id: td_ecomap
     task: draw_ecomap
+    partial:
+      tile_layers:
+        - name: "SATELLITE"
```
Comment (walljcg) [inline examples/subject-tracking/spec.yaml:423]: -name: "TERRAIN"
-name: "SATELLITE"
opacity: 0.5

File: examples/subject-tracking/spec.yaml [hunk 9]
```diff
@@ -377,6 +475,7 @@ workflow:
     task: create_plot_widget_single_view
     partial:
       data: ${{ workflow.nsd_chart_html_url.return }}
+      title: "NSD Chart"
```
Comment (walljcg) [inline examples/subject-tracking/spec.yaml:478]: "Net Square Displacement"

**Review by walljcg — CHANGES_REQUESTED** [review]
Looking great!! Couple of minor things.

File: examples/events/spec.yaml (hunk 1 above)
Comment (cisaacstern) [reply examples/events/spec.yaml]: resolved in [b55ef94](https://github.com/wildlife-dynamics/ecoscope-workflows/pull/389/commits/b55ef947141543f31f7911da4b9b3835ece53bb6)

File: examples/events/spec.yaml (hunk 2 above)
Comment (cisaacstern) [reply examples/events/spec.yaml]: resolved in [b55ef94](https://github.com/wildlife-dynamics/ecoscope-workflows/pull/389/commits/b55ef947141543f31f7911da4b9b3835ece53bb6)

File: examples/events/spec.yaml (hunk 3 above)
Comment (cisaacstern) [reply examples/events/spec.yaml]: resolved in [b55ef94](https://github.com/wildlife-dynamics/ecoscope-workflows/pull/389/commits/b55ef947141543f31f7911da4b9b3835ece53bb6)

File: examples/subject-tracking/spec.yaml (hunk 4 above)
Comment (cisaacstern) [reply examples/subject-tracking/spec.yaml:120]: resolved in [b55ef94](https://github.com/wildlife-dynamics/ecoscope-workflows/pull/389/commits/b55ef947141543f31f7911da4b9b3835ece53bb6)

File: examples/subject-tracking/spec.yaml (hunk 5 above)
Comment (cisaacstern) [reply examples/subject-tracking/spec.yaml]: resolved in [3fc9192](https://github.com/wildlife-dynamics/ecoscope-workflows/pull/389/commits/3fc91925c256859066250abe050c3ca34a55daa4)

File: examples/subject-tracking/spec.yaml (hunk 9 above)
Comment (cisaacstern) [reply examples/subject-tracking/spec.yaml]: resolved in [2f03ea8](https://github.com/wildlife-dynamics/ecoscope-workflows/pull/389/commits/2f03ea8fe44ab205b457daf834bb1b39a05c53f1)

File: examples/subject-tracking/spec.yaml (hunk 7 above)
Comment (cisaacstern) [reply examples/subject-tracking/spec.yaml]: resolved in [2f03ea8](https://github.com/wildlife-dynamics/ecoscope-workflows/pull/389/commits/2f03ea8fe44ab205b457daf834bb1b39a05c53f1)

File: examples/subject-tracking/spec.yaml (hunk 6 above)
Comment (cisaacstern) [reply examples/subject-tracking/spec.yaml]: resolved in [2f03ea8](https://github.com/wildlife-dynamics/ecoscope-workflows/pull/389/commits/2f03ea8fe44ab205b457daf834bb1b39a05c53f1)

File: examples/subject-tracking/spec.yaml (hunk 8 above)
Comment (cisaacstern) [reply examples/subject-tracking/spec.yaml]: resolved in [3fc9192](https://github.com/wildlife-dynamics/ecoscope-workflows/pull/389/commits/3fc91925c256859066250abe050c3ca34a55daa4)

**Comment by cisaacstern** [comment]
@walljcg thanks for the review! I believe all the requested fixes are in and all tests are passing. If this looks good to you we can merge and deploy, or lmk if you see any other changes we should make first. Thanks!

**Comment by cisaacstern** [comment]
Going to merge because the review items have been addressed and I'd like to get something up on gcp by my EOD for review.

**Pull request merged**